games/applications/dj.py

```diff
@@ -0,0 +1,97 @@
+"""Deustch Jozsa Benchmark Circuits"""
+
+
+import os
+import numpy as np
+from qiskit import QuantumCircuit
+import pytest
+from . import backends, run_qiskit_circuit
+
+
+QASM_DIR = os.path.join(os.path.dirname(os.path.abspath(__file__)), "qasm")
+NUM = 5
+# -----------------------------------------------------------------------------------
+
+
+def constant(n):
+    """Creates a constant oracle"""
+    qc = QuantumCircuit(n + 1)
+    output = np.random.randint(2)
+    if output == 1:
+        qc.x(n)
+    c_oracle = qc.to_gate()
+    c_oracle.name = "Slayyy Oracle"
```

Comment:
It might be better to give this a more descriptive name like `constant_oracle` that way we can distinguish the oracle in the qasm file.